doc/api/util.markdown

```diff
@@ -399,6 +414,7 @@ Returns `true` if the given "object" is a `Buffer`. `false` otherwise.
     util.isBuffer(new Buffer('hello world'))
       // true
 
+_Note: This function is deprecated._
 
 ## util.inherits(constructor, superConstructor)
```

Comment:
inherits is deprecated?
The entire planet uses it. What's the new preferred way (apart from ES6 classes)?